### ISSUE TITLE
[upd] Actionlint triggers to inputs count false positively

### ISSUE THREAD

**Issue by va4st** [rhysd/actionlint#600]
```[actionlint] reported by reviewdog 🐶 maximum number of inputs for "workflow_dispatch" event is 10 but 11 inputs are provided. see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs [events] Raw Output: e:.github/workflows/***: maximum number of inputs for "workflow_dispatch" event is 10 but 11 inputs are provided.```

However, if we're follow the link we can see that `The maximum number of top-level properties for inputs is 25 .`

**Comment by muzimuzhi** [comment]
Already fixed in dev but not released yet.
- #598
